examples/online-shop/service.bal

```diff
@@ -0,0 +1,149 @@
+listener http:Listener onlineShopListener = new (9090);
+
+service /onlineshop on onlineShopListener {
+    resource function get employees() returns Employee[]|error? {
+        Employee[] employees = [];
+         stream<Employee, error?> resultStream = dbClient->query(`SELECT * FROM EMPLOYEES`);
+         _ = check resultStream.forEach(function(Employee employee) {
+            employees.push(employee);
+         });
+         check resultStream.close();
+         return employees;
+    }
+
+    resource function get employees/[decimal id]() returns Employee|error? {
+        stream<Employee, error?> resultStream = dbClient->query(`SELECT * FROM EMPLOYEES WHERE EMPLOYEE_ID = ${id}`);
+        record {|Employee value;|}? data = check resultStream.next();
+        check resultStream.close();
+        return data?.value;
+    }
+
+    resource function get totalemployeesCount() returns int|error? {
+        return check dbClient->queryRow(`SELECT COUNT(*) FROM EMPLOYEES`, int);
+    }
+
+    resource function get products() returns Product[]|error? {
+        Product[] products = [];
+        stream<Product, error?> resultStream = dbClient->query(`SELECT * FROM PRODUCTS`);
+        _ = check resultStream.forEach(function(Product product) {
+           products.push(product);
+        });
+        check resultStream.close();
+        return products;
+    }
+
+    resource function get products/[decimal id]() returns Product|error? {
+        stream<Product, error?> resultStream = dbClient->query(`SELECT * FROM PRODUCTS WHERE PRODUCT_ID = ${id}`);
+        record {|Product value;|}? data = check resultStream.next();
+        check resultStream.close();
+        return data?.value;
+    }
+
+    resource function get customers() returns Customer[]|error? {
+        Customer[] customers = [];
+        stream<Customer, error?> resultStream = dbClient->query(`SELECT * FROM CUSTOMERS`);
+        _ = check resultStream.forEach(function(Customer customer) {
+           customers.push(customer);
+        });
+        check resultStream.close();
+        return customers;
+    }
+
+    resource function get customers/[decimal id]() returns Customer|error? {
+        stream<Customer, error?> resultStream = dbClient->query(`SELECT * FROM CUSTOMERS WHERE CUSTOMER_ID = ${id}`);
+        record {|Customer value;|}? data = check resultStream.next();
+        check resultStream.close();
+        return data?.value;
+    }
+
+    resource function get orders() returns Order[]|error? {
+        Order[] orders = [];
+        stream<Order, error?> resultStream = dbClient->query(`SELECT * FROM ORDERS`);
+        _ = check resultStream.forEach(function(Order order) {
+           orders.push(order);
+        });
+        check resultStream.close();
+        return orders;
+    }
+
+    resource function get orders/[decimal id]() returns Order|error? {
+        stream<Order, error?> resultStream = dbClient->query(`SELECT * FROM ORDERS WHERE ORDER_ID = ${id}`);
```

Comment:
`queryRow`?

Reply:
Use queryRaw in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/9c39326fb639c7553dd84fc179accd374e46f2ff